src/circuits/joinsplit.tcc

```diff
@@ -292,8 +353,7 @@ public:
 
             // [SANITY CHECK] Total size of unpacked inputs
             size_t total_size_unpacked_inputs = 0;
-            for (size_t i = 0;
-                 i < NumOutputs + NumInputs + 1 + 1 + 1 + NumInputs;
+            for (size_t i = 0; i < NumInputs + NumOutputs + 1 + NumInputs + 1;
```

Comment:
Use `nb_field_residual` instead of the final `+ 1` (applicable to other places in this file)